lib/aurora_bootstrapper/table.rb

```diff
@@ -44,8 +44,12 @@ def blacklisted_field?( field )
       end
     end
 
+    def timestamp
+      @timestamp ||= @client.query( 'select ( unix_timestamp() - (select variable_value from information_schema.global_status where variable_name = "uptime" )  ) as time;').first['time']
```

Comment:
let's backdate this like an hour, to account for the time it takes for aws to spin up the instance (which is hopefully in the minutes range, but it doesnt hurt to backdate an hour)